modules/control/testdata/conf/control_conf.pb.txt

```diff
@@ -102,6 +102,7 @@ lat_controller_conf {
     adaption_state_gain: 1.0
     adaption_desired_gain: 1.0
     adaption_nonlinear_gain: 1.0
+    adaption_matrix_p: 1.0
```

Comment:
@kevin-y-wang So default first order?

Reply:
In the testdata/conf, the default is first-order